tests/conftest.py

```diff
@@ -290,7 +306,7 @@ def default_2000_blocks_compact(bt, consensus_mode):
 def default_10000_blocks_compact(bt, consensus_mode):
     from tests.util.blockchain import persistent_blocks
 
-    if consensus_mode == ConsensusMode.SOFT_FORK4:
+    if consensus_mode == ConsensusMode.SOFT_FORK4 or consensus_mode == ConsensusMode.HARD_FORK_2_0:
```

Comment:
I'm not sure why I had to disable these tests. perhaps it takes a long time to generate these chains. I'll try to generate these over the weekend. This and `default_10000_blocks`